### PR TITLE
add more eclipse contents

### PR DESCRIPTION
**Pull request by daniel-sol**

### CODE REVIEW

**Comment by perolavsvendsen** [comment]
Schema must also be updated with the added contents

File: schema/definitions/0.8.0/schema/fmu_results.json [hunk 1]
```diff
@@ -271,7 +271,12 @@
                         "khproduct",
                         "timeseries",
                         "wellpicks",
-                        "parameters"
+                        "parameters",
+                        "relperm",
+                        "rft",
+                        "pvt",
+                        "lift_curves",
+                        "transissibilities"
```
Comment (perolavsvendsen) [inline schema/definitions/0.8.0/schema/fmu_results.json:279]: typo

**Review by perolavsvendsen — CHANGES_REQUESTED** [review]
See comments, suggest:
- Shorten (if possible) lists of parameters etc in summary table example, to make it readable.
- Typo in "transmissibilities" (or however this is spelled)

**Review by perolavsvendsen — APPROVED** [review]
LGTM 🎸 

@jcrivenaes comments?

**Comment by perolavsvendsen** [comment]
Probably better to Squash and merge, not Rebase and merge

**Comment by jcrivenaes** [comment]
> Probably better to Squash and merge, not Rebase and merge

Why? I prefer rebase and merge, after user do manual squash if needed

**Comment by perolavsvendsen** [comment]
> > Probably better to Squash and merge, not Rebase and merge
> 
> Why? I prefer rebase and merge, after user do manual squash if needed

Sure, yes. But only rebase/merge (default) will put all the commits into history.

**Pull request merged**